<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>743</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>16</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="myLabel" name="pictureLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>256</width>
     <height>291</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>117</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>333333</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>upload</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restartPushButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>50</y>
     <width>256</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>restart</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="resultTextEdit">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>290</y>
     <width>256</width>
     <height>235</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myLabel</class>
   <extends>QLabel</extends>
   <header location="global">mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
